helm-system-packages-dnf.el

```diff
@@ -27,8 +27,16 @@
 ;;; Code:
 (require 'helm-system-packages)
 
+(require 'seq)
```

Comment:
Seq was introduced in Emacs 25.1 so you should change the requirements in the header.

Reply:
`seq` is part of Emacs core since 25.1, you are right. Nevertheless, `seq` also exists as separate package compatible with Emacs 24: https://git.savannah.gnu.org/cgit/emacs/elpa.git/tree/packages/seq.

Reply:
OK.